docs/deployment/prerequisites.md

```diff
@@ -13,9 +13,15 @@ This guide assumes that you have:
    - [python](https://www.python.org/downloads/) - A programming language used for automated installation scripts.
    - [pip](https://pip.pypa.io/en/stable/installation/) - A package installer for python.
    
-1. Created an EKS cluster
+1. Creating an EKS cluster
     - If you do not have an existing cluster, run the following command to create an EKS cluster. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
-    - Substitute values for the CLUSTER_NAME and CLUSTER_REGION in the script below
+    - Various controllers in Kubeflow deployment use IAM roles for service accounts(IRSA). An OIDC provider must exist for your cluster to use IRSA.
+        - **Note:** If you are using an existing cluster, Create an OIDC provider and associate it with for your EKS cluster by running the following command:
```

Comment:
lets move the note to after the create cluster command so the continuity does not break

and make exporting the cluster_name and region separate step so that it can be used in either command